Eatery/Controllers/Eateries/Campus/CampusEateriesViewController.swift

```diff
@@ -16,6 +18,8 @@ class CampusEateriesViewController: EateriesViewController {
 
     private var preselectedEateryName: String?
 
+    var networkActivityIndicator: NVActivityIndicatorView?
```

Comment:
can this be `private`?

Reply:
Short answer: no

Because the Eateries Shared View Controller has access to the navigation bar, and the network activity indicator goes there. So the `EateriesSharedVC` has to assign the activity indicator to the `CampusEateriesVC`.